src/include/txn_inline.h

```diff
@@ -435,6 +436,12 @@ __wt_txn_op_delete_commit_apply_timestamps(WT_SESSION_IMPL *session, WT_TXN_OP *
     txn = session->txn;
     page_del = ref->page_del;
 
+    if (F_ISSET(op->btree, WT_BTREE_LOGGED))
```

Comment:
Could you mention something about these checks?

Reply:
Added comments.